client/src/pages/LyricsView/LyricsDisplay.tsx

```diff
@@ -52,50 +78,116 @@ const LyricsDisplay: React.FunctionComponent<IProps> = ({
+  const toggleSyncEnabled = () => {
+    setSyncEnabled(s => !s);
+  };
 
   return (
     <div className={classes.root}>
-      {/* TODO https://material-ui.com/components/text-fields/#input-adornments */}
-      {searchShown ? (
-        <Box mb={1}>
-          <TextField
-            variant="outlined"
-            value={search}
-            onChange={onUserSearch}
-            InputProps={{
-              endAdornment: (
-                <InputAdornment position="end">
-                  <IconButton onClick={toggleSearchShown} edge="end">
-                    <CloseIcon />
-                  </IconButton>
-                </InputAdornment>
-              )
-            }}
-            label="Search"
-            placeholder="Search lyrics you heard to find your position..."
-            style={{ width: "100%", maxWidth: 600 }}
-          />
-        </Box>
-      ) : (
-        <IconButton className={classes.toggleSearchButton} onClick={toggleSearchShown}>
-          <SearchIcon fontSize="small" />
-        </IconButton>
-      )}
+      <Toolbar className={classes.toolbar}>
+        {/* TODO https://material-ui.com/components/text-fields/#input-adornments */}
+        {searchShown ? (
+          <Box mb={1}>
+            <TextField
+              variant="outlined"
+              value={search}
+              onChange={onUserSearch}
+              InputProps={{
+                endAdornment: (
+                  <InputAdornment position="end">
+                    <IconButton onClick={toggleSearchShown} edge="end">
+                      <CloseIcon />
+                    </IconButton>
+                  </InputAdornment>
+                )
+              }}
+              label="Search"
+              placeholder="Search lyrics you heard to find your position..."
+              style={{ width: "100%", maxWidth: 600 }}
+            />
+          </Box>
+        ) : (
+          <IconButton onClick={toggleSearchShown}>
+            <SearchIcon fontSize="small" />
+          </IconButton>
+        )}
+        {syncPossible ? (
+          <IconButton onClick={toggleSyncEnabled}>
+            {syncEnabled ? <SyncEnabledIcon /> : <SyncDisabledIcon />}
+          </IconButton>
+        ) : (
+          <SyncDisabledIcon />
+        )}
+      </Toolbar>
       {lyrics ? (
         <div>
           <Typography component="div" className={classes.lyrics} ref={lyricsRef}>
-            {lyrics}
+            {syncedLyrics.before}
+            <br />
+            <span id="highlighted-text" className={classes.mark} ref={highlightedRef}>
+              {syncedLyrics.highlighted}
+            </span>
+            <br />
+            {syncedLyrics.after}
           </Typography>
-          <Box mt={2} textAlign="left">
-            <Typography>
-              <Link href={`https://genius.com${geniusUrl}`}>
-                Lyrics for {lyricsTitle} by {lyricsArtist}
-              </Link>
-            </Typography>
-          </Box>
```

Comment:
I would be keen to keep this, but maybe the text could just be "Lyrics attribution" and the link can be sourced from the backend.

For example, instead of `geniusUrl` that we had before (still a left over reference in `client/src/hooks/useLyrics.ts`), we could call it `lyricsAttributionHref`. For Genius lyrics, we set it to `https://genius.com${geniusUrl}` on the server side. For lrclib.net, we can set it to `https://lrclib.net/`.

We're using this content for free so I'm really happy to let users know where it's coming from 😀